ieee.csl

```diff
@@ -262,6 +280,18 @@
       <text variable="locator"/>
     </group>
   </macro>
+  <macro name="location">
```

Comment:
Location is included in the macros for conference presentations, but the representation is different for online videos. It's probably worth moving the location information from those other macros, and making this a dedicated location macro that handles all cases.